app/code/core/Mage/Contacts/controllers/IndexController.php

```diff
@@ -92,19 +91,22 @@ public function postAction()
                     );
 
                 if (!$mailTemplate->getSentSuccess()) {
-                    throw new Exception();
+                    Mage::throwException($this->__('Unable to submit your request. Please, try again later'));
```

Comment:
I would not use a comma after the word Please, because it doesn't look good. It would mean adding a comma in the phrase above "Please submit your request".